sdk-docs/getting-started/extension-points.md

```diff
@@ -5,44 +5,45 @@ parentDocSlug: getting-started
 ---
 
 > 🚧 Beta
-> 
+>
 > This is a beta version and subject to change without notice. Pricing, terms, conditions and availability may change in the final version.
 
-An Iris App is a collection of a number of extensions. 
+An Iris App is a collection of a number of extensions.
 
 ![](https://cdn.statically.io/gh/trackunit/developer-hub/master/iris%20app.png)
 
-The Iris App SDK provides these extension points in the manager. 
+The Iris App SDK provides these extension points in the manager.
 
 # Asset Home Extension Point
 
 This extension point allows you to add a new tab to the Assets Home screen within Trackunit Manager, as illustrated in the image below. The extension is controlled in the extension-manifest.ts file.
 
 [block:image]
 {
-  "images": [
-    {
-      "image": [
-        "https://cdn.statically.io/gh/trackunit/developer-hub/master/sdk-docs/extension_to_asset_home.png",
-        null,
-        "Asset home extension"
-      ],
-      "align": "center",
-      "sizing": "50% "
-    }
-  ]
+"images": [
+{
+"image": [
+"https://cdn.statically.io/gh/trackunit/developer-hub/master/sdk-docs/extension_to_asset_home.png",
+null,
+"Asset home extension"
+],
+"align": "center",
+"sizing": "50% "
+}
+]
```

Comment:
Any reason to de indent this?

Reply:
no reason, looks like its a sneaky work of my formatter (i think? ). I have added back all the indentation now :)